packages/orbit-components/src/Timeline/TimelineStep/index.jsx

```diff
@@ -48,25 +45,24 @@ const TimelineStep = ({ children, label, subLabel, type, asText }: Props): React
   if (isDesktop)
     return (
       <TimelineStepDesktop
-        typeIcon={<TypeIcon type={type} />}
         nextType={nextType}
+        prevType={prevType}
         label={label}
-        asText={asText}
+        last={last}
+        active={isActive}
         type={type}
         subLabel={subLabel}
-        last={last}
       >
         {children}
       </TimelineStepDesktop>
     );
 
   return (
     <TimelineStepMobile
-      typeIcon={<TypeIcon type={type} />}
       nextType={nextType}
       label={label}
-      asText={asText}
```

Comment:
If we're removing this prop this is a breaking change, no?

Reply:
yes, mentioned that in the PR description

Reply:
Oh, I missed that. I don't know for sure if we can/should trust the tracking, because as it is open source we never know for sure who else might be using it, and so we should respect the breaking change. But I'll leave the decision to you.

Reply:
why? the purpose of the tracking exact for that reason, to make decisions about breaking changes

Reply:
kinda agree about the open source, but for us the product is prio, so we should more care about that. I do not want to block that changes, because in case if we would decide to release it with other breaking changes, it would have to wait a couple of weeks 🤔